source/fundamentals/linq.txt

```diff
@@ -609,6 +608,35 @@ The following is a subset of the documents contained in the returned results:
    { "_id" : ObjectId(...), "name" : "Abaleh", "restaurant_id" : "50009096", "cuisine" : "Mediterranean", "address" : { ... }, "borough" : "Manhattan", "grades" : [...] }
    ...
 
+.. _csharp-linq-convert:
+
+$convert
+~~~~~~~~
+
+You can use the ``Mql.Convert()`` method in a ``Select()`` projection to
```

Comment:
I can see from the other operators in the page that usually we first introduce briefly the operator, and then show how to use it in LINQ. Maybe we should follow that pattern?

Reply:
I moved some sections around to distinguish between stages/operators